ext/curl/tests/curl_version_features-array.phpt

```diff
@@ -0,0 +1,34 @@
+--TEST--
+Test curl_version() - feature_list functionality
+--EXTENSIONS--
+curl
+--FILE--
+<?php
+    $info_curl = curl_version();
+    var_dump(array_key_exists("feature_list", $info_curl));
+    var_dump(!array_is_list($info_curl['feature_list']));
+
+    echo implode(', ', array_keys($info_curl['feature_list']));
+    echo PHP_EOL;
```

Comment:
```suggestion
    print_r(array_map(fn ($v) => get_debug_type($v), $info_curl['feature_list']));
```

would be better and will test individual value types too

Reply:
There are some features that depend on the libcurl version, so I tried to avoid having to inspect the whole array. But yeah you are right it makes more sense to directly compare like this. I went ahead and added a check for this as well, but with a `%A` match which leaves room for version-dependent features in this array too.

Reply:
I agree. I also meant to replace the l8-l12, as the implode is covered by the new dump.

Reply:
I see, thank you. I removed those duplicates. Thank you!